aten/src/ATen/cuda/CUDABlas.cpp

```diff
@@ -214,6 +215,36 @@ static size_t _getWorkspaceSize() {
   return workspace_size;
 }
 
+void* _getWorkspaceWithoutHandle() {
+  cublasHandle_t handle = at::cuda::getCurrentCUDABlasHandle();
+  auto stream = c10::cuda::getCurrentCUDAStream();
+  cudaStream_t _stream = stream;
+  auto key = std::make_tuple(static_cast<void *>(handle), static_cast<void *>(_stream));
+  auto workspace_it = at::cuda::cublas_handle_stream_to_workspace().find(key);
+  TORCH_CHECK(workspace_it != at::cuda::cublas_handle_stream_to_workspace().end());
+  return workspace_it->second.mutable_get();
+}
+
+void* _getWorkspace(size_t& workspaceSize) {
+  #ifdef USE_ROCM
+    // See https://github.com/pytorch/pytorch/issues/73328 for reasoning behind
+    // setting this to 1M.
+    workspaceSize = _getWorkspaceSize();
+  #else
+    workspaceSize = at::cuda::getChosenWorkspaceSize();
```

Comment:
this would still change behavior compared to previous?

Reply:
good catch, missed that!